include/mxnet/executor.h

```diff
@@ -152,19 +152,19 @@ class Executor {
   static Executor* SimpleBind(nnvm::Symbol symbol,
```

Comment:
@reminisce: I'm not sure that I'd consider this a breaking API change.  My understanding from what was discussed  the dev list was that we would follow the hour-glass model for the c_api, and make the c_api the only native interface that we semantically version.

Reply:
@KellenSunderland You got a valid point. My worry was that since this function is declared in the header placed in the `include` directory, we have no idea whether users have used this function or not to build something for their own.

Reply:
@reminisce Yes I do see what you mean.  It's certainly possible someone used the header.  As long as it's not too much work would you mind creating a second function @mkolod?

Reply:
I think we don't need to maintain compatibility on this. Its unlikely any user would depend on it.

Reply:
Thanks @piiswrong !

Reply:
@mkolod As we discussed offline about not breaking existing APIs, could you create a simple bind API for you to use only, rather than modifying this one?